modular_skyrat/master_files/code/modules/pain_shock/shock_helper.dm

```diff
@@ -0,0 +1,233 @@
+#define COMSIG_SHOCK_UPDATE "shock_update"
+
+/* /mob/living/carbon/Life(delta_time = SSMOBS_DT, times_fired)
+	flow_control() */
+/mob/living/carbon
+	var/lastpainmessage
+	var/in_shock = FALSE
+	var/flow_rate = BASE_FLOW_RATE
+	var/injuries
+/mob/living/carbon/Initialize(mapload)
+	. = ..()
+	RegisterSignal(src, COMSIG_SHOCK_UPDATE, .proc/shock_helper)
```

Comment:
I don't see why this should be a signal that isn't sent by shock_helper itself
🤔